crates/holochain_conductor_api/src/docs.rs

```diff
@@ -10,7 +10,7 @@
 //!
 //! For example:
 //!
-//! ```
+//! ```ignore
```

Comment:
Did this use to be doc tested successfully?

Reply:
I think this changed with the rust version bump. But either way, these aren't tests, so shouldn't be compiled.